src/NzbDrone.Common/Instrumentation/CleanseLogMessage.cs

```diff
@@ -20,6 +20,7 @@ public class CleanseLogMessage
                 new Regex(@"getnzb.*?(?<=\?|&)(r)=(?<secret>[^&=]+?)(?= |&|$)", RegexOptions.Compiled | RegexOptions.IgnoreCase),
                 new Regex(@"(?<=authkey = "")(?<secret>[^&=]+?)(?="")", RegexOptions.Compiled | RegexOptions.IgnoreCase),
                 new Regex(@"(?<=beyond-hd\.[a-z]+/api/torrents/)(?<secret>[^&=][a-z0-9]+)", RegexOptions.Compiled | RegexOptions.IgnoreCase),
+                new Regex(@"(?<=blutopia\.[a-z]+/torrent/download/\d+\.)(?<secret>[^&=][a-z0-9]+)", RegexOptions.Compiled | RegexOptions.IgnoreCase),
```

Comment:
This won't work... this format is used on all UNIT3D download links. Not just blutopia. Lets make it more generic

Reply:
Other than blutopia and needing to generify it, any ideas why the regex isn't work yet works on the regex tester?

Reply:
fixed regex; second test case added

regex worked fine; test case was invalid,  since corrected